application/engine/services/autopilot_daemon.py

```
@@ -688,16 +695,55 @@ async def _handle_writing(self, novel: Novel):
-                await self.chapter_workflow.post_process_generated_chapter(
+                post = await self.chapter_workflow.post_process_generated_chapter(
                     novel.novel_id.value, chapter_num, outline, chapter_content, scene_director=None
                 )
+                chapter_content = post.get("content") or chapter_content
+                seam_rewrite_info = post.get("seam_rewrite_info") or {}
+                await self._upsert_chapter_content(novel, next_chapter_node, chapter_content, status="draft")
                 logger.info(f"[{novel.novel_id}]    ✅ post_process_generated_chapter 完成")
+                if getattr(self.chapter_workflow, "_requires_manual_seam_revision", None) and \
+                        self.chapter_workflow._requires_manual_seam_revision(seam_rewrite_info):
+                    logger.warning(
+                        "[%s] 第 %s 章接缝复检未通过，进入人工修订: attempts=%s status=%s",
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)